packages/mdc-textfield/mdc-text-field.scss

```diff
@@ -183,7 +183,11 @@
 }
 
 .mdc-text-field--fullwidth {
-  @include mdc-text-field-full-width_;
+  @include mdc-text-field-fullwidth_;
+}
+
+.mdc-text-field--fullwidth.mdc-text-field--invalid {
```

Comment:
When the alternate colors class is added, the idle line is still red.

Reply:
FWIW I would point out that the alternate color class was never doing anything to the idle bottom line color for the fullwidth text field, so this might actually be a separate issue, but I'm looking into it a bit.

Reply:
So this seems like it was more of an oversight with the demo page than with these styles. I added styles to the demo page for the full-width custom color demo, and also fixed a style for the textfield custom color demo that seemed accidentally redundant (idle/focus used same opacity).

I suspect that those styles could be further cleaned up; I initially ended up confused as to where to put the styles for this specific part of the demo page, and realized that the custom color and custom error color styles probably don't actually need separate CSS classes since they're always applied together to the same elements...